### PR TITLE
Task cinema seats booking

### PR DESCRIPTION
**Pull request by ik-web**
##HTML movie seat booking

[Demo](https://ik-web.github.io/html-movie-seat-booking/)

Please, review!

### CODE REVIEW

File: submissions/ik-web/html-movie-seat-booking/index.html [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+        <section class="movie-info">
+            <h1 class="movie-info__title">Monster Hunter</h1>
+            <div class="movie-info__inner">
+                <div class="movie-info__poster">
+                    <img src="img/movie.jpg" alt="movie poster">
+                </div>
+
+                <ul class="movie-info__about">
+                    <li class="movie-info__about-item about-item__format">3D</li>
+                    <li class="movie-info__about-item about-item__map">Kyiv, SEC "Lavina", Hall 13</li>
```
Comment (A-Ostrovnyy) [inline submissions/ik-web/html-movie-seat-booking/index.html]: For address use HTML tag `addres` [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address)

File: submissions/ik-web/html-movie-seat-booking/index.html [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+                    <li class="movie-info__about-item about-item__date">2021.02.04 Thursday</li>
```
Comment (A-Ostrovnyy) [inline submissions/ik-web/html-movie-seat-booking/index.html]: For time/date use HTML tag `time`  [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time)

File: submissions/ik-web/html-movie-seat-booking/index.html [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+            <h2 class="booking-seats__title">Choose your seats</h2>
+
+            <div class="booking-seats__cinema-hall">
+                <div class="cinema-hall__screen">screen</div>
+                <form class="cinema-hall__seats" id="seats-form">
+
+                    <ol class="cinema-hall__row">
+                        <li>
+                            <label>
+                                <input type="checkbox" class="cinema-hall__check" value="1">
```
Comment (A-Ostrovnyy) [inline submissions/ik-web/html-movie-seat-booking/index.html:45]: Add styles on `:hover` and `:focus` states.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comments below.)

File: submissions/ik-web/html-movie-seat-booking/index.html (hunk 1 above)
Comment (ik-web) [reply submissions/ik-web/html-movie-seat-booking/index.html]: I fixed this point, thanks for the comment!

File: submissions/ik-web/html-movie-seat-booking/index.html (hunk 2 above)
Comment (ik-web) [reply submissions/ik-web/html-movie-seat-booking/index.html]: I fixed this moment

File: submissions/ik-web/html-movie-seat-booking/index.html (hunk 3 above)
Comment (ik-web) [reply submissions/ik-web/html-movie-seat-booking/index.html:45]: Added. Thank you for this comment!

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**